src/compiler.js

```diff
@@ -52,6 +52,7 @@ class Compiler {
         return true
       }
     } catch (e) {
+      console.log(e)
```

Comment:
Do we want this log message? _If_ we log anything here, it should be done using the `print` module of the Gluegun toolbox.

Reply:
I think usually this should never happen, I hit it due to a double variable declaration while developing. Changed this to use gluegun.

Reply:
Ah, yeah, any errors during compilation should already be logged inside the `try {}` branch.